### DISCUSSION TITLE
New Radiation Pattern

### DISCUSSION THREAD

**Discussion by Adheidari** [NVlabs/sionna#389] [Q&A]
Hello 

I'm a PhD student working on Sionna ray-tracing model. As of my understanding allows, right now we can use tr38.901 radiation pattern which goes back to release 14. I was wondering whether I can generate other patterns relevant to later releases. And I'll be grateful if you can provide me some documentation regarding this problem.

Regards,

**Comment by jhoydis** [comment]
Hi @Adheidari,

Yes, you can very easily define new antenna patterns. As described [here](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Antenna), an antenna pattern is simply a callable that takes two vectors as arguments containing the elevation and azimuth angles for which the pattern shall be evaluated. You can checkout the source code of the available patterns to see how they are implemented.

Is there a specific pattern from a later release that you have in mind?

**Comment by Adheidari** [comment]
Thanks for your quick response. 
I was curious how I can add an specific antenna pattern relevant to my work to the simulation like PIFA for UE.
